src/Parser/Shaman/Elemental/Modules/Talents/LightningRod.js

```diff
@@ -21,21 +20,33 @@ class LightningRod extends Analyzer {
     combatants: Combatants,
   };
 
+  lightningRodDamage=0;
+
   on_initialized() {
     this.active = this.combatants.selected.hasTalent(SPELLS.LIGHTNING_ROD_TALENT.id);
   }
 
+  on_byPlayer_damage(event) {
+    if (event.ability.guid===SPELLS.LIGHTNING_ROD_DAMAGE.id){
+      this.lightningRodDamage+=event.amount;
+    }
+  }
+
   get rawUpdate() {
     return this.enemies.getBuffUptime(SPELLS.LIGHTNING_ROD_DEBUFF.id) / this.owner.fightDuration;
   }
 
   statistic() {
     return (
-      <StatisticBox
-        icon={<SpellIcon id={SPELLS.LIGHTNING_ROD_TALENT.id} />}
-        value={`${formatPercentage(this.rawUpdate)} %`}
-        label={'Uptime'}
-      />
+      <StatisticBox icon={<SpellIcon id={SPELLS.LIGHTNING_ROD_TALENT.id} />}
+        value={(
+          <span>
+            {`${formatPercentage(this.rawUpdate)} %`}<br/>
+            {formatNumber(this.lightningRodDamage)}{' '}
```

Comment:
It's completely inconsistent with style to show raw damage rather than damage as a percentage of total. I assume you're doing this so the player doesn't mix up the uptime and the damage?

It's more typical for spells like this to show the damage alone in the statistic and then the uptime in the tooltip (or vice versa).